File: count_the_age_person/count_the_age_person.cpp
using namespace std;

#include <iostream>

int main()
{
	int BirthMonth, BirthYear;
	int CurrentMonth, CurrentYear;
	int AgeYear, AgeMonth;
	
	cout<<"\n\n\t\t\tCount the age person\n\n";
	cout<<"Enter Your Birth Year(Eg:1989):";
	cin>>BirthYear;
	
	cout<<"\n\nEnter Your Birth Month(Eg:7):";
	cin>>BirthMonth;
	
	cout<<"\nEnter The Current Month(Eg:7):";
	cin>>CurrentMonth;
	
	cout<<"\nEnter The Current Year(Eg:2010):";
	cin>>CurrentYear;
	
	AgeYear=CurrentYear-BirthYear;
	AgeMonth=12-BirthMonth;
	
	cout<<"\n\n\t\tYour Age is "<<AgeYear<<" Years And "<<AgeMonth<<" Months ";
	
	return 0;
}

File: count_the_age_person/count_the_age_person.py
print ("\t\t*******Count the age person********\n\n")
birthYear = int(input("Enter Your Birth Year(Eg:1989): "))
birthMonth = int(input("\nEnter Your Birth Month(Eg:7): "))
currentYear = int(input("\nEnter The Current Year(Eg:2016): "))
currentMonth = int(input("\nEnter The Current Month(Eg:7): "))

ageYear = currentYear - birthYear
ageMonth = currentMonth - birthMonth

if ageMonth == 0:
    print ("\n\n\t\tYour age is", ageYear, "Years")

elif ageMonth == 1:
    print ("\n\n\t\tYour age is", ageYear, "Years And",ageMonth,"Month")
    
else:
    print ("\n\n\t\tYour age is", ageYear, "Years And",ageMonth,"Months")
